tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -238,18 +238,19 @@ stateMachine params = iterate (\s -> stateFor s.next) CLIENT_START_STATE
         CLIENT_START_STATE = state 1 [
             action clientHello client True,
             action changeCipherSpec client (params.zero_rtt /\ compatMode client)
-        ] (if params.retry then HELLO_RETRY_STATE else SERVER_HELLO_STATE)
+        ] (if (params.retry == 1) then HELLO_RETRY_STATE else SERVER_HELLO_STATE)
 
         HELLO_RETRY_STATE = state 2 [
-            action helloRetryRequest server True,
+            action serverHello server True,
             action changeCipherSpec server (compatMode server),
-            action changeCipherSpec client (~params.zero_rtt /\ compatMode client),
-            action clientHello client True
-        ] SERVER_HELLO_STATE
+            action changeCipherSpec server (~params.zero_rtt /\ compatMode server),
```

Comment:
is this right? From the RFC, I don't see a restriction on server for zero rtt:  https://tools.ietf.org/html/rfc8446#appendix-D.4
```
      The server sends a dummy change_cipher_spec record immediately
      after its first handshake message.  This may either be after a
      ServerHello or a HelloRetryRequest.
```